editor.planx.uk/src/@planx/components/MapAndLabel/Public/index.tsx

```diff
@@ -227,6 +227,8 @@ const Root = () => {
     previouslySubmittedData,
   } = mapAndLabelProps;
 
+  const [passport] = useStore((state) => [state.computePassport()]);
```

Comment:
fyi: If we're just grabbing a single value, we can use the following notation - 

```suggestion
  const passportData = useStore((state) => state.computePassport().data);
```